### PR TITLE
lesson03 - modules

### PR DESCRIPTION
**Pull request by sortigoza**
adds: lesson03 - modules

lesson03/README.md contains some notes for the slides.

Some changes from lesson02 (in the module):

- using now locals for app_name, appversion to keep it simple
- removed data for availability_zones since we are using vpc_zone_identifier, otherwise it crashes
- using "${local.app_name}-elb" for elb name instead of an independent variable

Create prod env
```
ENV=prod ./terraform init
ENV=prod ./terraform apply
```

 Create dev env
```
ENV=dev ./terraform init
ENV=dev ./terraform apply
```

### CODE REVIEW

File: lesson03/environments/terraform-dev.tfvars [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+env = "dev"
+
+metadata = {
+  appname = "academy-user-N"
+  appversion = "latest"
+}
+
+tags = {
+  project = "academy-user-N"
```
Comment (chitopunk) [inline lesson03/environments/terraform-dev.tfvars:11]: parametrized

File: lesson03/environments/terraform-prod.tfvars [hunk 2]
```diff
@@ -0,0 +1,16 @@
+aws_region = "us-east-2"
+
+env = "prod"
+
+metadata = {
+  appname = "academy-user-N"
```
Comment (chitopunk) [inline lesson03/environments/terraform-prod.tfvars:6]: same here

File: lesson03/environments/terraform-prod.tfvars [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  appversion = "latest"
+}
+
+tags = {
+  project = "academy-user-N"
```
Comment (chitopunk) [inline lesson03/environments/terraform-prod.tfvars:11]: here parameter

**Review by chitopunk — CHANGES_REQUESTED** [review]

**Comment by sortigoza** [comment]
parametrized in https://github.com/wizelineacademy/terraform-academy/pull/13/files#diff-c0f726443ff58cba193c56b9b72afb0cR35

file lesson03/environments/terraform, line 35

File: lesson03/environments/terraform-prod.tfvars (hunk 3 above)
Comment (sortigoza) [reply lesson03/environments/terraform-prod.tfvars:11]: see comment below

File: lesson03/environments/terraform-prod.tfvars (hunk 2 above)
Comment (sortigoza) [reply lesson03/environments/terraform-prod.tfvars:6]: see comment below

File: lesson03/environments/terraform-dev.tfvars (hunk 1 above)
Comment (sortigoza) [reply lesson03/environments/terraform-dev.tfvars:11]: see comment below

**Review by dancortes94 — APPROVED** [review]
LGTM

**Review by chitopunk — APPROVED** [review]
LGTM

**Pull request merged**